ghost.js

```diff
@@ -0,0 +1,24 @@
+var Ghost = function(){
+  var randy = Math.floor(Math.random()*4 + 1);
+  console.log (randy);
+  switch (randy){
+    case 1:
+    this.color = "red";
+    break;
+
+    case 2:
+    this.color = "pink";
+    break;
+
+    case 3:
+    this.color = "white";
+    break;
+
+    default:
+    this.color = "yellow";
+  }
+}
+
+Ghost.prototype.color = function(color){
```

Comment:
I do not see any added functionality with this part. However, the rest looks good :+1: 

Please update your PR to remove the this part, and I'll go ahead and accept it.